substrate/frame/revive/src/exec.rs

```diff
@@ -731,9 +735,9 @@ where
 			value,
 			debug_message,
 		)? {
-			stack.run(executable, input_data)
+			stack.run(executable, input_data).map(|_| stack.first_frame.last_frame_output)
 		} else {
-			Self::transfer_no_contract(&origin, &dest, value)
+			Self::transfer_no_contract(&origin, &dest, value).map(|_| Default::default())
```

Comment:
Doesn't this already return `ExecResult`? No map necessary.

Reply:
Yep good catch, fixed.